### PR TITLE
added "#note" to readme.md

### PR DESCRIPTION
**Pull request by JJDUK**
TLDR: added a note into README.md

Done:
Edited the README.md file
This modifies line 2 to read "#note" (no real reason, other than to practice Git commands)

To Do:

- [ ]  Review my PR
- [ ]  Merge PR and Delete feature branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # GitHub for Developers
-
+#note
```
Comment (SHills96) [inline README.md:2]: this is incredible.

Comment (SHills96) [inline README.md:2]: Incredible stuff.

**Review by SHills96 — APPROVED** [review]
Fantastic edit. This will change the game.

File: README.md (hunk 1 above)
Comment (mk3jamie) [reply README.md:2]: I seen that change and I agree, game changer!

**Pull request merged**